Source/Core/VideoCommon/RenderBase.h

```diff
@@ -53,9 +53,17 @@ struct EfbPokeData
   u32 data;
 };
 
-// TODO: Move these out of here.
 extern int frameCount;
-extern int OSDChoice;
+
+enum class OSDMessage : s32
+{
+  IR_Changed = 1,
+  AR_Toggled = 2,
+  EFBCopy_Toggled = 3,
+  Fog_Toggled = 4,
+  Speed_Changed = 5,
+  XFB_Changed = 6
```

Comment:
Why is it "EFB Copy toggled" but "XFB changed"? The code suggests it could be "XFB toggled" instead.

Reply:
Because it's actually shown for multiple settings and isn't just toggling a single one. Although that might not be a perfect reason for naming it "Changed".